### DISCUSSION TITLE
[Help] Is there a possibility to search inside a table

### DISCUSSION THREAD

**Discussion by mpgr04** [fomantic/Fomantic-UI#1594] [Collections (form, grid, menu, message, table, ...)] [answered]
# Help Wanted
I just wondered if there currently is a possibility to perform search operations on table content?
I've seen the search module and i wonder if i can somehow combine it with the records inside my table?

**Comment by exoego** [comment]
Do you mean filtering table content by search/input?
Table module does not have filtering feature.
I think it is required to write JS codes to initialize table content and filter them by input.

**Comment by lubber-de** [comment, marked as answer]
I suggest to look at https://github.com/DataTables/DataTables
It has support for usage of Fomantic/Semantic classes to style tables and has lots of plugins, also for searching/filtering

https://datatables.net/examples/styling/semanticui.html
https://datatables.net/plug-ins/filtering/

**Comment by lubber-de** [comment]
Closing because using the `search` module to adjust any table is something we won't implement into the core and external libraries like datatables are doing such job already quite well.
